app/dashboard/_components/RiskAndOpps/RiskOppCard.tsx

```diff
@@ -0,0 +1,23 @@
+import Card from '@/components/Card'
+import { risksData } from '@/app/dashboard/_components/RiskAndOpps/data/RisksData'
+import { opportunitiesData } from '@/app/dashboard/_components/RiskAndOpps/data/OpportunitiesData'
+
+const RiskOppCards = ({ type }) => {
+  const data = type === 'risks' ? risksData : opportunitiesData
+
+  return (
+    <div className='grid grid-cols-1 gap-4 md:grid-cols-2'>
+      {data.map((item) => (
+        <Card
+          key={item.title}
+          classname='md:col-span-6 col-span-3 rounded-8 mt-4 gap-2'
```

Comment:
since you only have declared the grid to have 1 or 2 colums col-span-6 / 3 doesn't work here. rounded is already declared on the card component, no need to override it. gap-2 won't do anything here since its not a grid or flexbox. i assume the mt-4 is causing double whitespace here since you already have a gap on the containing div